### DISCUSSION TITLE
Ansible lint flags documented molecule converge file syntax as invalid

### DISCUSSION THREAD

**Discussion by JohnMalmberg** [ansible/ansible-lint#1603] [Q&A]
This is a follow up to https://github.com/ansible-community/ansible-lint/issues/1379

As requested, I asked for the correct syntax in the molecule community.  There has been no response for a documented alternative for CI use of molecule in Ansible.

What I have found since is that the syntax that Ansible Lint is flagging as invalid is actually documented in Molecule as the syntax to use.

https://molecule.readthedocs.io/_/downloads/en/stable/pdf/  See section 3.2.5.

Since this is actually in the Molecule documentation, it should pass Ansible Lint, or some supported alternative is needed.

**Comment by ssbarnea** [comment]
Please read https://github.com/ansible-community/ansible-lint/pull/1606 -- shortly even molecule playbooks should pass ansible syntax check. They need to be updated in order to assure ansible syntax check is passing on them.
